codes/quantum/qubits/ShorCode.yml

```diff
@@ -0,0 +1,55 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: shor_code
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Shor''s code'
+
+# First reference with the code (optional).
+introduced: 'Shor''s code is the first known quantum error correction code. \cite{doi:10.1103/PhysRevA.52.R2493}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Shor''s code, also known as the [[9,1,3]] code, is an instance of degenerate \hyperref[code:stabilizer]{stabillizer code}. Shor'' code is also an exmaple of planar code \cite{arxiv:quant-ph/9810055}.'
```

Comment:
Make surface a cousin with detail: 'Shor's code is the smallest surface code defined on the projective plane \cite{arxiv:quant-ph/9810055}.'